build.sbt

```diff
@@ -533,7 +547,7 @@ lazy val readme =
 lazy val `native-settings` = Seq(
     fork                                        := false,
     bspEnabled                                  := false,
-    libraryDependencies += "io.github.cquiroz" %%% "scala-java-time" % "2.6.0" % "provided"
+    libraryDependencies += "io.github.cquiroz" %%% "scala-java-time" % "2.6.0"
```

Comment:
Unrelated change: I was testing scripts in Scala native with scala-cli and they fail with the missing Java time classes